Mathlib/Topology/Algebra/Order/Field.lean

```diff
@@ -135,6 +135,11 @@ theorem Filter.Tendsto.div_atTop {a : 𝕜} (h : Tendsto f l (𝓝 a)) (hg : Ten
   simp only [div_eq_mul_inv]
   exact mul_zero a ▸ h.mul (tendsto_inv_atTop_zero.comp hg)
 
+lemma tendsto_const_div_atTop {F ι: Type*} [TopologicalSpace F] [LinearOrderedField F]
+    [OrderTopology F] (p : Filter ι) (g : ι → F) (r : F) (hg : Tendsto g p atTop) :
+    Tendsto (fun n ↦ r / g n) p (𝓝 0) := by
+  apply Filter.Tendsto.div_atTop tendsto_const_nhds hg (f := fun _ => r) (a := r)
```

Comment:
You should change the name to make use of dot notation. Also, you should use all the existing variables in this file instead of writing them from scratch. Also, neither `p` nor `g` should not be explicit. Finally, your proof is providing Lean with a lot more information than it needs.
```suggestion
lemma Filter.Tendsto.const_div_atTop (hg : Tendsto g l atTop) (r : 𝕜)  :
    Tendsto (fun n ↦ r / g n) l (𝓝 0) :=
  tendsto_const_nhds.div_atTop hg
```

Reply:
thanks!